contentcuration/contentcuration/settings.py

```diff
@@ -240,7 +240,10 @@
     pycountry.LOCALES_DIR,
 )
 
-ugettext = lambda s: s
```

Comment:
I assume this is a PEP8 thing?  but why?

Reply:
Not sure, probably a whitespace thing...